app/assets/javascripts/controllers/report_data_controller.js

```diff
@@ -341,10 +341,14 @@
     if (mainContent) {
       angular.element(mainContent).removeClass('miq-sand-paper');
       angular.element(mainContent).removeClass('miq-list-content');
+      angular.element(document.querySelector('#paging_div .miq-pagination')).css('display', 'none');
+      console.log(document.getElementById('paging_div'));
       if (viewType && (viewType === 'grid' || viewType === 'tile')) {
+        angular.element(document.querySelector('#paging_div .miq-pagination')).css('display', 'block');
```

Comment:
I did not realized that document should not be used based on eslint rules. Can you please change it to
```javascript
angular.element(this.$document.querySelector('#paging_div .miq-pagination')).css('display', 'block');
```
And couple of lines bellow
```javascript
angular.element(this.$document.querySelector('#paging_div .miq-pagination')).css('display', 'block');
```